WORKSPACE

```diff
@@ -107,6 +107,24 @@ git_repository(
     shallow_since = "1582757372 -0800",
 )
 
+http_archive(
+    name = "rules_proto",
+    sha256 = "4d421d51f9ecfe9bf96ab23b55c6f2b809cbaf0eea24952683e397decfbd0dd0",
+    strip_prefix = "rules_proto-f6b8d89b90a7956f6782a4a3609b2f0eee3ce965",
+    urls = [
+        "https://github.com/bazelbuild/rules_proto/archive/f6b8d89b90a7956f6782a4a3609b2f0eee3ce965.tar.gz",
+    ],
+)
+
+http_archive(
+    name = "io_bazel_rules_closure",
```

Comment:
this is an unsupported ruleset and a very unfortunate dependency. do you understand why it's required? does the runtime depend on closure library? does it depend on `goog.base` for the module loading?

Reply:
Yeah, its required only so that I can load in the com_google_protobuf workspace. Which I only grab for the protoc plugin. If you feel strongly I can potentially remove it since its not actually a dependency of the protoc plugin